### PR TITLE
Notification management

### PR DESCRIPTION
**Pull request by ATGE**

### CODE REVIEW

File: SoftLayer/CLI/user/edit_notifications.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+from SoftLayer.CLI import environment
+
+
+@click.command()
+@click.option('--enable/--disable', default=True,
+              help="Enable (DEFAULT) or Disable selected notification")
+@click.argument('notification', nargs=-1, required=True)
+@environment.pass_env
+def cli(env, enable, notification):
+    """Enable or Disable specific notifications.
```
Comment (allmightyspiff) [inline SoftLayer/CLI/user/edit_notifications.py:16]: Mention that notification names need quotes around them, something like this.
`Enable or Disable specific notifications for the active user. Notification names should be enclosed in quotation marks.`

File: SoftLayer/CLI/user/notifications.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command()
+@environment.pass_env
+def cli(env):
+    """User Notifications."""
```
Comment (allmightyspiff) [inline SoftLayer/CLI/user/notifications.py:12]: Change this from `User Notifications` to `My Notifications`.

File: SoftLayer/CLI/user/edit_notifications.py (hunk 1 above)
Comment (ATGE) [reply SoftLayer/CLI/user/edit_notifications.py]: Done

**Comment by allmightyspiff** [comment]
LGTM

File: SoftLayer/CLI/user/notifications.py (hunk 2 above)
Comment (ATGE) [reply SoftLayer/CLI/user/notifications.py]: Done.

**Pull request merged**